pkg/download/bucket/bucket.go

```diff
@@ -107,6 +114,16 @@ func convertObject(o []byte, projectName string) (config.Config, error) {
 		return config.Config{}, fmt.Errorf("variable %q unreadable", bucketName)
 	}
 
+	// skip unmodifiable buckets
+	if u, ok := r.Get(updatable).(bool); ok && !u {
```

Comment:
I find the error pretty weird too tbh. Wouldn't it be nicer to just partially deserialize the object (like we do for automation) and look at the updatable field - instead of looking and throwing an error on `convertObject` if it's un-modifiable? (just writing that makes it seem odd for 'convert' to fail if the object is unmodifiable)

Reply:
Yes, I agree, it's exactly my comment above. 
We have multiple checks, and moving them one level above also seems funny. Do you have a concrete idea on how we could solve this?

Reply:
It is; that's why I added the original comment to the PR. 
Moving the checks out also seems strange: We move it one level up with many continue statements or one level up and down into a new validation function - where we still need to differentiate between error and skip. Do you have a concrete solution for how we can do this nicely? I'd like to do it nicely :D